mkosi/__init__.py

```diff
@@ -4266,7 +4258,7 @@ def extract_partition(
 def gen_kernel_images(args: MkosiArgs, root: Path) -> Iterator[Tuple[str, Path]]:
     # Apparently openmandriva hasn't yet completed its usrmerge so we use lib here instead of usr/lib.
     for kver in root.joinpath("lib/modules").iterdir():
-        if not (kver.is_dir() and os.path.isfile(os.path.join(kver, "modules.dep"))): # type: ignore
```

Comment:
I had to drop the modules.dep check here since we haven't run depmod yet when running kernel-install here since we disable running kernel-install as part of package installation. If this is still a problem, we'll need to figure out another way to distinguish valid directories from nonvalid directories